.github/workflows/pr-checks.yaml

```diff
@@ -90,44 +85,57 @@ jobs:
+        uses: ./.github/actions/setup-cli
+
+      - name: Acceptance tests
+        timeout-minutes: 60
         run: |
-          curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v${SDK_VERSION}/operator-sdk_linux_amd64
-          chmod +x operator-sdk
-          mv -v operator-sdk $GITHUB_WORKSPACE/bin/
+          eval $(minikube docker-env)
+          make OPERATOR_REPO_REF=$(minikube ip):5000/sbo SKIP_REGISTRY_LOGIN=true release-operator -o registry-login test-acceptance-with-bundle
 
-          curl -Lo opm https://github.com/operator-framework/operator-registry/releases/download/v${OPM_VERSION}/linux-amd64-opm
-          chmod +x opm
-          mv -v opm $GITHUB_WORKSPACE/bin/
 
-          curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v${K8S_VERSION}/bin/linux/amd64/kubectl
-          chmod +x kubectl
-          mv -v kubectl $GITHUB_WORKSPACE/bin/
+  acceptance_non_olm:
+    name: Non OLM Acceptance Tests with Kubernetes
+    runs-on: ubuntu-20.04
 
-          curl -Lo minikube https://storage.googleapis.com/minikube/releases/v${MINIKUBE_VERSION}/minikube-linux-amd64
-          chmod +x minikube
-          mv -v minikube $GITHUB_WORKSPACE/bin/
+    env:
+      EXTRA_BEHAVE_ARGS: "--tags=~@knative --tags=~@openshift --tags=~@olm"
+      UMOCI_VERSION: "0.4.5"
+      OPERATOR_VERSION: "0.4.0"
 
-      - name: Set KUBECONFIG
-        run: |
-          KUBECONFIG=$GITHUB_WORKSPACE/miniconfig
-          echo "KUBECONFIG=$KUBECONFIG" >> $GITHUB_ENV
-          touch $KUBECONFIG
+    steps:
+      - name: Checkout Git Repository
+        uses: actions/checkout@v2
 
-      - name: Set up minikube
-        run: |
-          ./hack/start-minikube.sh start --kubernetes-version=v${K8S_VERSION} --driver=$CONTAINER_RUNTIME --cpus $(nproc) --memory 5g
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "^1.15.6"
 
-      - name: Docker Info
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+          architecture: "x64"
+
+      - name: Set up CLI
+        uses: ./.github/actions/setup-cli
+
+      - name: Setup umoci cli
         run: |
-          eval $(minikube docker-env)
-          docker ps
-          kubectl get nodes -o yaml
-          kubectl cluster-info
-          docker info
+          curl -Lo umoci https://github.com/opencontainers/umoci/releases/download/v${UMOCI_VERSION}/umoci.amd64
+          chmod +x umoci
+          mv -v umoci $GITHUB_WORKSPACE/bin/
 
-      - name: Acceptance tests
+      - name: Acceptance tests with release artifacts
         timeout-minutes: 60
         run: |
           eval $(minikube docker-env)
-          make OPERATOR_REPO_REF=$(minikube ip):5000/sbo SKIP_REGISTRY_LOGIN=true release-operator -o registry-login test-acceptance-with-bundle
\ No newline at end of file
+          GIT_COMMIT_ID=$(git rev-parse --short=8 HEAD)
```

Comment:
you do not need to set this here - it is already set in `Makefile`.

Reply:
I tried it without the GIT_COMMIT_ID mentioned here and the CI is not picking up from the Makefile

Reply:
Can you be more specific? Can you share the job run?

Reply:
./hack/prepare-operatorhub-pr.sh 0.4.0 192.168.49.2:5000/sbo:bundle-0.4.0- 
time="2021-02-16T16:30:13Z" level=fatal msg="Error initializing source docker://192.168.49.2:5000/sbo:bundle-0.4.0-: Error reading manifest bundle-0.4.0- in 192.168.49.2:5000/sbo: manifest unknown: manifest unknown"

Reply:
https://github.com/redhat-developer/service-binding-operator/pull/806/checks?check_run_id=1912133716

Reply:
You should have the line like this:

https://github.com/redhat-developer/service-binding-operator/pull/806/files#diff-7a6a4578fe04c4fc86b28e2dc9672e48d1a9f721315e540b90bac0ef12263053R95

in order to things to work.